### PR TITLE
Update dependencies version

### PR DESCRIPTION
**Pull request by code-guru**
It seems updating packages dependencies specifily `vinyl-sourcemaps-apply` will solve some bugs.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,11 +29,11 @@
   },
   "dependencies": {
     "clean-css": "^3.3.3",
-    "gulp-util": "^3.0.5",
+    "gulp-util": "^3.0.7",
```
Comment (shinnn) [inline package.json]: Not needed. https://nodesource.com/blog/semver-tilde-and-caret

File: package.json [hunk 2]
```diff
@@ -29,11 +29,11 @@
...
     "object-assign": "^4.0.1",
-    "readable-stream": "^2.0.0",
+    "readable-stream": "^2.0.4",
```
Comment (shinnn) [inline package.json]: Not needed. https://nodesource.com/blog/semver-tilde-and-caret

**Comment by shinnn** [comment]
Only `vinyl-sourcemaps-apply` update is meaningful and all the other changes are not needed.

**Comment by shinnn** [comment]
@code-guru Any updates?

**Comment by code-guru** [comment]
As you already pointed out minor version changes are automatically get
updated in npm. Since I had problem with gulp minify i thought this may
help. Do you want me to corrext the pr and send it again?

**Comment by shinnn** [comment]
> Do you want me to corrext the pr and send it again?

Yes, please fix *this PR*, but don't create a new PR. You can update #127 by adding more commits and squash them.

**Comment by shinnn** [comment]
ping @code-guru

**Comment by shinnn** [comment]
`vinyl-sourcemaps-apply` update is included in v1.2.2.

@code-guru Thanks for trying to contribute to this project.

**Pull request closed without merging**